<launch>
  <arg name="robot_ns" default="uav" />

  <group ns="$(arg robot_ns)">
    <node pkg="aerial_robot_simulation" type="mujoco_ros_control" name="mujoco_ros_control" output="screen"/>

    <node pkg="aerial_robot_simulation" type="mujoco_flight_control_interface.py" name="mujoco_flight_controller" output="screen"/>
  </group>

</launch>
